js/index.js

```diff
@@ -3,29 +3,47 @@
 function updateSubtotal(product) {
   console.log('Calculating subtotal, yey!');
 
-  //... your code goes here
+  const price = product.querySelector('.price span');
```

Comment:
```suggestion
  const price = product.querySelector('.price span').innerText;
```
Desde aqui directamente puedes obtener el resultado